d2l-input-time.html

```diff
@@ -0,0 +1,392 @@
+				<input
+					id="{{id}}-combobox"
+					type="text"
+					class="d2l-input"
+					role="combobox"
+					aria-autocomplete="list"
+					aria-label$="{{label}}"
+					aria-owns$="{{listboxId}}"
+					aria-activedescendant$={{selectedListboxId}}
+					on-focus="_onTimeInputFocused"
+					value="{{value::input}}">
+			</div>
+
+			<d2l-dropdown-content
+					id="dropdown"
+					min-width="[[_dropdownWidth]]"
+					max-width="[[_dropdownWidth]]"
+					boundary='[[boundary]]'
+					align="start"
+					no-auto-close
+					no-pointer
+					no-auto-focus
+					no-padding
+					on-tap="_handleTimesListClick">
+
+					<div class="dropdown-content">
+						<iron-selector id="{{listboxId}}" role="listbox" selected-attribute="selected">
+							<template is="dom-repeat" items="{{times}}">
+								<div id="{{listboxId}}-{{index}}" class="d2l-input-time-item" role="option">{{item}}</div>
+							</template>
+						</iron-selector>
+					</div>
+				</d2l-dropdown-content>
+		</d2l-dropdown>
+		<span class="timezone">{{timezone}}</span>
+	</template>
+
+	<script>
+	/* global d2lIntl */
+	(function() {
+		var instances = 0;
+		Polymer({
+
+			is: 'd2l-input-time',
+
+			behaviors: [
+				Polymer.IronA11yKeysBehavior
+			],
+
+			properties: {
+				timeInterval: {
+					type: Number,
+					value: 30
+				},
+				hours: {
+					type: Number,
+					notify: true,
+					value: 23
+				},
+				minutes: {
+					type: Number,
+					notify: true,
+					value: 59
+				},
+				value: {
+					type: String,
+					notify: true,
+					observer: '_valueChanged'
+				},
+				locale: {
+					type: String,
+					value: ''
+				},
+				overrides: {
+					type: Object,
+					value: {}
+				},
+				timezone: String,
+				formatter: {
+					type: Object,
+					readOnly: true,
+					value: new d2lIntl.DateTimeFormat()
+				},
+				parser: {
+					type: Object,
+					readOnly: true,
+					value: new d2lIntl.DateTimeParse()
+				},
+				times: {
+					type: Object,
+					readOnly: true,
+					value: []
+				},
+				boundary: Object,
+				_dropdownWidth: String,
+				listboxId: String,
+				selectedListboxId: String,
+				label: String
+			},
+
+			observers: [
+				'_updateFormater(locale, overrides)',
+				'updateValue(hours, minutes)',
+				'updateSelection(hours, minutes)',
+				'_updateTimes(timeInterval, formatter)'
+			],
+
+			listeners: {
+				'iron-select': '_scrollSelectedItemIntoView'
+			},
+
+			keyBindings: {
+				'esc': '_onTimeInputKeyPressed',
+				'enter': '_onTimeInputKeyPressed',
+				'up': '_onTimeInputKeyPressed',
+				'down': '_onTimeInputKeyPressed',
+				'home': '_onTimeInputKeyPressed',
+				'end': '_onTimeInputKeyPressed'
+			},
+
+			clear: function() {
+				this.updateValue(23, 59);
+			},
+
+			// time display logic start
+
+			_updateFormater: function(locale, overrides) {
+				this._setFormatter(new d2lIntl.DateTimeFormat(locale, { locale: overrides }));
+				this._setParser(new d2lIntl.DateTimeParse(locale, { locale: overrides }));
```

Comment:
It's in [this issue](https://github.com/BrightspaceUI/time-picker/issues/32), but we should really switch this over to use the localize behavior's time/date formatting methods at some point.

Reply:
Added https://github.com/BrightspaceUI/inputs/issues/19